dataland-frontend/src/components/resources/newLandingPage/EmailButton.vue

```diff
@@ -0,0 +1,22 @@
+<template>
```

Comment:
Question regarding the styling of buttons:
The "I AM INTERESTED" button looks exactly the same as e.g. the signup button or the "Get in Touch" button.
So could we define the styling of these buttons only once globally, and then reuse it in the components, instead of implementing a button design for each and every segment of the landing page?

Reply:
Yes, that's already planned when the style-guide will be approved and final. Same for some other elements like H2, H3, icons and so on. I'll take care of the buttons first, let me ask for the final approve on them, especially the hover state

Reply:
All right sounds great.  Will that happen in another Pull Request?

Reply:
I created this story for it:

https://jira.d-fine.dev/browse/DALA-2636

Is it ok?  Feel free to add to its description or title.

If it is ok for you, I will close this conversation.